docs/docs/modules/indexes/vector_stores/integrations/supabase.mdx

```diff
@@ -57,5 +59,18 @@ $$;
 
 import CodeBlock from "@theme/CodeBlock";
 import Example from "@examples/indexes/vector_stores/supabase.ts";
+import MetadataFilterExample from "@examples/indexes/vector_stores/supabase_with_metadata_filter.ts";
+
+### Standard Usage
+
+The below example shows how to perform a basic similarity search with Supabase:
 
 <CodeBlock language="typescript">{Example}</CodeBlock>
+
+### Metadata Filtering
+
+Given the above `match_documents` Postgres function, you can also pass a filter parameter to only documents with a specific metadata field value.
+
+**Note:** If you've previously been using `SupabaseVectorStore`, you may need to create a separate Postgres function with a different name and an additional `filter jsonb DEFAULT '{}'` parameter to use this functionality.
```

Comment:
Reword as something like "... you may need to drop and recreate the `match_documents` function` per the updated sql above"